src/controllers/document.js

```diff
@@ -136,10 +136,15 @@ class DocumentController extends BaseController {
     const request = {
       index,
       collection,
-      body: {ids},
-      action: 'mGet'
+      action: 'mGet',
+      includeTrash: options.includeTrash
```

Comment:
This options does not exists in Kuzzle v2 anymore, it was related to the trashcan which have been removed